modules/reindex/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -20,4 +20,6 @@
 grant {
    // reindex opens socket connections using the rest client
    permission java.net.SocketPermission "*", "connect";
+   // rest client uses system properties which gets the default proxy
+   permission java.net.NetPermission "getProxySelector";
```

Comment:
I don't understand why you have to give reindex the permission rather than to the ` grant codeBase "${codebase.elasticsearch-rest-client-6.0.0-beta1-SNAPSHOT.jar}" {` codebase (as well as ` grant codeBase "${codebase.httpasyncclient-4.1.2.jar}" {`?  you can do that in this file as well